pkg/cloudclient/aws/private.go

```diff
@@ -125,35 +130,35 @@ func (c *Client) validateInstanceType(ctx context.Context) error {
 	// Describe the provided instance type only
 	//      https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ec2#DescribeInstanceTypesInput
 	descInput := ec2.DescribeInstanceTypesInput{
-		InstanceTypes: []ec2Types.InstanceType{ec2Types.InstanceType(c.instanceType)},
+		InstanceTypes: []ec2Types.InstanceType{ec2Types.InstanceType(c.clientInput.ClientConfig.AWSConfig.InstanceType)},
 	}
 
-	c.logger.Debug(ctx, "Gathering description of instance type %s from EC2", c.instanceType)
+	//c.logger.Debug(ctx, "Gathering description of instance type %s from EC2", c.clientInput.InstanceType)
```

Comment:
Is this a useful debug-level log, or no? If not, I say remove it completely instead of leaving it commented out

Reply:
I'm opting not to remove existing logs unless the refactor calls for it. But it's a good consideration to do a cleanup of logs at some point.

Reply:
uncommented